tests/test_tools.py

```diff
@@ -14,11 +14,128 @@ def test_angular_avg_nd_3():
     P = np.repeat(P, 100).reshape(400, 400, 100)
     freq = [x, x, np.linspace(-2, 2, 100)]
     p_k, k_av_bins = angular_average_nd(P, freq, bins=50, n=2)
+    p_k_lin, k_av_bins_lin = angular_average_nd(
+        P, freq, bins=50, n=2, interpolation_method="linear"
+    )
```

Comment:
instead of doing this, put a `@pytest.mark.parameterize('interpolation_method', [None, 'linear'])` on the top of the test